network-api/networkapi/buyersguide/templates/about/how_to_use.html

```diff
@@ -1,9 +1,11 @@
 {% extends "../bg_about.html" %}
 
-{% load i18n %}
+{% load static i18n %}
 
 {% block main_about_class %}how-to-use{% endblock %}
 
+{% block catgory_nav %}{% endblock %}
```

Comment:
Why are we declaring this block in every file? (I don't see it in bg_about, nor bg_page)